Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Build the manager binary
 FROM golang:1.15 as builder
 
+ARG GoLinkerArgs
```

Comment:
This argument will need to be specified also in CI.
It would be easier to pass only the validator image URL and tag, instead of generic linker arguments.